settings.gradle

```diff
@@ -26,7 +26,9 @@ buildCache {
 
     remote(gradleEnterprise.buildCache) {
         enabled = true
-        push = isCI
+        // Check access key presence to avoid build cache errors on PR builds when access key is not present
+        def accessKey = System.getenv("GRADLE_ENTERPRISE_ACCESS_KEY")
+        push = isCI && accessKey != null && !accessKey.isEmpty()
```

Comment:
This can be simplified to isCi && accessKey.

Reply:
updated

Reply:
But please be sure to verify it and not take my word for it.

Reply:
Yup, this PR is from a fork, and there are no 403 errors or other remote build cache errors in the logs: https://github.com/gradle/common-custom-user-data-gradle-plugin/actions/runs/7003812403/job/19050268047